lib/workers/pr/changelog/source-github.js

```diff
@@ -60,9 +60,10 @@ async function getChangeLogJSON({
     host: host === 'github.com' ? 'api.github.com' : host,
   });
   if (!config) {
-    logger.debug('Repository URL does not match any hnown hosts');
+    logger.debug('Repository URL does not match any known hosts');
     return null;
   }
+  const githubApiBaseURL = config.endpoint;
```

Comment:
Does this still work for GHE when you need to fetch release notes from github.com? I may be misunderstanding it but it seems to be hardcoding the API lookup to be off GHE when most packages would be looked up from github.com still.

Reply:
When debugging, this appears to be set to `'https://api.github.com` when the release notes are github.com, which is great. Originally I had done it a different way where I was comparing the url to see if it was from the same repository as the `RENOVATE_ENDPOINT`, etc, but this made the change much simpler.

Reply:
I need to dig into why that is and also try to reduce the chance we don't break it in future :)